CMakeLists.txt

```diff
@@ -507,20 +504,25 @@ set(scylla_sources
     replica/memtable.cc
     message/messaging_service.cc
     multishard_mutation_query.cc
-    mutation.cc
-    mutation_fragment.cc
-    mutation_partition.cc
-    mutation_partition_serializer.cc
-    mutation_partition_view.cc
-    mutation_query.cc
+    mutation/atomic_cell.cc
+    mutation/canonical_mutation.cc
+    mutation/frozen_mutation.cc
+    mutation/mutation.cc
+    mutation/mutation_fragment.cc
+    mutation/mutation_partition.cc
```

Comment:
mutation_partition_v2.cc should be moved too

Reply:
It was never in CMakeLists.txt. I'll add it.